template/src/main/twirl/ch.epfl.scala.index.views/project/badges.scala.html

```diff
@@ -13,14 +16,14 @@ <h2>Create badge</h2>
       <a class="close" href="#">&times;</a>
 
       <div class="content">
-        <img src="@release.reference.badgeUrl" />
+        <img src="@release.badgeUrl(env)" />
         <div>
           <h5>Markdown:</h5>
           <pre
           aria-label="Badge markdown"
           id="copy-badge-markdown"
           class="copyable-incantation"
-          >[![@release.reference.artifact Scala version support](@release.reference.badgeUrl)](@release.reference.artifactFullHttpUrl)</pre>
+          >[![@release.artifactName Scala version support](@release.badgeUrl(env)](@release.artifactFullHttpUrl(env)</pre>
```

Comment:
Two small typos introduced here as only the opening bracket is introduced with `(env`! Fix in https://github.com/scalacenter/scaladex/pull/786